tfunc/tfunc.go

```diff
@@ -17,10 +17,24 @@ func All() template.FuncMap {
 	return all
 }
 
-func Env() template.FuncMap {
+// Consul querying functions
+func ConsulV0() template.FuncMap {
```

Comment:
For the functions in this file, since they're public, the comments should start with function name

Reply:
Addressed these and the few others that needed the same tweak.